### PR TITLE
Kaushal erp bugfix

### PR DESCRIPTION
**Pull request by PITPL-Kaushal-Khute**
> Please provide enough information so that others can review your pull request:

> Explain the **details** for making this change. What existing problem does the pull request solve?

> Screenshots/GIFs

### CODE REVIEW

File: erpnext/selling/doctype/customer/customer.js [hunk 1]
```diff
@@ -201,6 +201,69 @@ frappe.ui.form.on("Customer", {
...
+			callback: function (r) {
+				price_list_rate = (r.message[0].price_list_rate);
+				var item_code = r.message[0].name
+				if (price_list_rate && price_list_rate != v.selling_price) {
+					frappe.confirm(
+						`Do you want to update the price do item ${v.item_name} of ${price_list_rate} for ${v.selling_price} in the price list  ?`, //ask something to update price
+						function () {
+							frappe.db.set_value("Item Price", item_code, "price_list_rate", v.selling_price)
+						},
+						function () {
```
Comment (PITPL-Akshay-Prajapati) [inline erpnext/selling/doctype/customer/customer.js:250]: don't added any code in this function. if unnecessary remove it.

File: erpnext/selling/doctype/customer/customer.py [hunk 2]
```diff
@@ -24,6 +25,8 @@
 	validate_party_accounts,
 )
 from erpnext.utilities.transaction_base import TransactionBase
+from pymysql import NULL
```
Comment (PITPL-Akshay-Prajapati) [inline erpnext/selling/doctype/customer/customer.py:28]: is not required remove it

File: erpnext/selling/doctype/customer/customer.py [hunk 3]
```diff
@@ -24,6 +25,8 @@
...
+from traitlets import Undefined
```
Comment (PITPL-Akshay-Prajapati) [inline erpnext/selling/doctype/customer/customer.py:29]: is not required remove it

**Review by PITPL-Akshay-Prajapati — CHANGES_REQUESTED** [review]

File: erpnext/stock/doctype/batch/batch.js [hunk 4]
```diff
@@ -162,3 +162,12 @@ frappe.ui.form.on('Batch', 'manufacturing_date', function (frm){
 		}
 	})
 })
+
+//Expiry Date Cannot Be Select Before Manufacturing Date
+frappe.ui.form.on('Batch', 'validate', function(frm) {
+
```
Comment (PITPL-Akshay-Prajapati) [inline erpnext/stock/doctype/batch/batch.js:168]: remove unnecessary blank lines.

**Review by PITPL-Akshay-Prajapati — APPROVED** [review]

File: erpnext/stock/doctype/batch/batch.js (hunk 4 above)
Comment (PITPL-Kaushal-Khute) [reply erpnext/stock/doctype/batch/batch.js:168]: ok, i will change

**Pull request merged**